kedro-org/kedro-viz
# [Fix] Change Types on Metadata Side Panel

## Description

resolves #752 . 

## Development notes

I have added a function to translate the naming of the different types of nodes on the FE to return the desired naming. ( The BE still returns the different types of node as `task`, `data` and `params`, while we want to ensure the nodes are displayed as `node`, `dataset` and `params`. 

I have also updated a variable naming ( I have kept `isTaskNode` as it is to avoid confusion for `isNodeNode`), as well as any related tests. 

## QA notes

The node type on the metadata panel should have the newly updated node type naming. 

## Checklist



<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/852"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
